### ISSUE TITLE
Semantic heading hierarchy

### ISSUE THREAD

**Issue by AdamJ** [AdamJ/TimeTrackerPro#75]
Several pages use `<h1>` classnames on <div> elements, which screen readers will not announce as landmarks.

## ToDo

- [ ] Change page titles to use actual heading elements
